packages/grafana-ui/src/components/Combobox/Combobox.tsx

```diff
@@ -157,7 +171,7 @@ export const Combobox = <T extends string | number>(props: ComboboxProps<T>) =>
       return null;
     }
 
-    if (value === null) {
+    if (!value) {
```

Comment:
How should we manage `''`? Should we treat it the same way as an undefined value? If you have an actual option with the value `''`, this won't work (imagine having a "default" case).